src/Serilog.Sinks.BrowserHttp/Serilog.Sinks.BrowserHttp.csproj

```diff
@@ -1,16 +1,16 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
     <RootNamespace>Serilog</RootNamespace>
-    <VersionPrefix>1.0.0</VersionPrefix>
+    <VersionPrefix>2.0.0</VersionPrefix>
```

Comment:
I thought this was appropriate, but it's up to you, of course!

Reply:
Since there hasn't yet been a stable 1.0 release of this package I think we can stick with the 1.x versioning - might be getting close to a stable version, though!

Reply:
Oh yes, I forgot it's all prerelease so far!